### PR TITLE
[ioctl] feat: Added Accountlist to newcmd

### LINKED ISSUE #2937: [ioctl] build `account list` command line into new ioctl
The new ioctl (./ioctl/newcmd) is a refactored version of the old ioctl(./ioctl/cmd). It includes two major changes:
- The [Client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) is passed when constructing the Cobra command.
- Unit tests are added for each command.

The code is a draft. The code of old ioctl can be used for reference.

**What would you like to be added**:
add `accountlist.go` and `accountlist_test.go` in `./ioctl/newcmd/account`. 
**Code for reference**:
https://github.com/iotexproject/iotex-core/blob/master/ioctl/cmd/account/accountlist.go

### PR DESCRIPTION
**Pull request by sajanjswl**
* Added accountlist to newcmd
* Added unit-test
* fix #2937

### CODE REVIEW

**Review by Liuhaai — APPROVED** [review]

**Comment by Liuhaai** [comment]
thx for contribution

File: ioctl/newcmd/account/accountlist.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+// Multi-language support
+var (
+	listCmdShorts = map[config.Language]string{
+		config.English: "List existing account for ioctl",
+		config.Chinese: "列出ioctl中已存在的账户",
+	}
+	listCmdUses = map[config.Language]string{
+		config.English: "list",
+		config.Chinese: "list",
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountlist.go:32]: delete if no Chinese translation is available. English is the fallback language

File: ioctl/newcmd/account/accountlist.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	"fmt"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-address/address"
+
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/cmd/alias"
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountlist.go:19]: "newcmd" will eventually replace "cmd", so don't import anything in "cmd"

File: ioctl/newcmd/account/accountlist.go [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	use, _ := c.SelectTranslation(listCmdUses)
+	short, _ := c.SelectTranslation(listCmdShorts)
+	al := &cobra.Command{
+		Use:   use,
+		Short: short,
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			listmessage := listMessage{}
+			aliases := alias.GetAliasMap()
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountlist.go:47]: copy GetAliasMap into "ioctl.Client", and rename it to "Alias"

File: ioctl/newcmd/account/accountlist_test.go [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	}()
+	ks := keystore.NewKeyStore(testAccountFolder,
+		keystore.StandardScryptN, keystore.StandardScryptP)
+	_, _ = ks.NewAccount("test")
+	_, _ = ks.NewAccount("test2")
+	client.EXPECT().NewKeyStore(gomock.Any(), gomock.Any(), gomock.Any()).Return(ks)
+
+	cmd := NewAccountList(client)
+	_, err := util.ExecuteCmd(cmd)
+	require.NoError(t, err)
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountlist_test.go:42]: cover the case in which "err" is not nil

Comment (sajanjswl) [reply ioctl/newcmd/account/accountlist_test.go]: all the comments have been addressed please review @CoderZhi

**Review by CoderZhi — APPROVED** [review]
thanks!

**Pull request merged**